addons/mail/static/src/models/call_main_view.js

```diff
@@ -149,6 +136,28 @@ registerModel({
                 showOverlayTimer: { doReset: this.showOverlayTimer ? true : undefined },
             });
         },
+        _updateLayout() {
+            if (!this.exists()) {
+                return;
+            }
```

Comment:
Each `on` method should have this guard.
It's a lie to update layout if you don't because of this.